website/docs/docs/partnerships.mdx

```diff
@@ -99,14 +99,14 @@ d) Example provider for reference: [AWS](https://github.com/terraform-providers/
 
 e) Contributing to Terraform [guidelines](https://github.com/hashicorp/terraform/blob/master/.github/CONTRIBUTING.md)
 
-f) HashiCorp developer [forum](https://discuss.hashicorp.com/c/terraform-providers/tf-plugin-sdk/43)
+f) HashiCorp Developer Discuss [forum](https://discuss.hashicorp.com/c/terraform-providers/tf-plugin-sdk/43)
 
 Please submit questions or suggestions about the Terraform SDK and provider development to the HashiCorp Terraform plugin SDK forum. If you are new to provider development and would like assistance, you can leverage one of the following development agencies that have developed Terraform providers in the past:
 
 | Partner            | Email                       | Website                                             |
 | ------------------ | :-------------------------- | :-------------------------------------------------- |
 | Crest Data Systems | <malhar@crestdatasys.com>   | [www.crestdatasys.com](http://www.crestdatasys.com) |
-| DigitalOnUs        | <hashicorp@digitalonus.com> | [www.digitalonus.com](http://www.digitalonus.com)   |
+| Tech Mahindra      | <hashicorp@digitalonus.com> | [www.digitalonus.com](http://www.digitalonus.com)   |
```

Comment:
@gautambaghel do we need to update the email and website too?

Reply:
Unfortunately no @laurapacilio, that remains the same. It's odd.

Reply:
huh, okay then! Just checking :)